### PR TITLE
feat: fixing the template.md link

### PR DESCRIPTION
**Pull request by zoesteinkamp**
closes #19599
This fixes the template.md file, originally it tried to search for it on influxdb, but now it routes to github properly

### CODE REVIEW

File: cypress/e2e/communityTemplates.test.ts [hunk 1]
```diff
@@ -211,6 +211,8 @@ describe('Community Templates', () => {
     it.skip('takes you to github readme when you click on the Community Templates button', () => {
       cy.getByTestID('community-template-readme-overlay-button').click()
       cy.get('.markdown-format').should('contain', 'Setup Instructions')
+
+      //we should confirm the link works
```
Comment (hoorayimhelping) [inline cypress/e2e/communityTemplates.test.ts:215]: what's this? is this a legit todo?

File: src/templates/utils/index.test.ts [hunk 2]
```diff
@@ -239,4 +240,14 @@ describe('validating template URLs', () => {
       ).toBe(TEMPLATE_URL_VALID)
     })
   })
+
+  describe('validating template readme Formatter', () => {
```
Comment (hoorayimhelping) [inline src/templates/utils/index.test.ts:244]: might consider adding another example, maybe with some text below the `## Setup Instructions`.

**Review by hoorayimhelping — APPROVED** [review]
great work!

File: cypress/e2e/communityTemplates.test.ts (hunk 1 above)
Comment (zoesteinkamp) [reply cypress/e2e/communityTemplates.test.ts:215]: should have added a better note, but yes its to check the link works and leads to the proper template.md file. Though we have this as a unit test, i guess we should do it in the e2e possibly

**Pull request merged**